### PR TITLE
make MultivariateNormalNP Samplable

### LINKED ISSUE #30: class MultivariateNormalNP should be Samplable
seems that other NPs are Samplable, maybe MultivariateNormalNP should also have the sample function?

### PR DESCRIPTION
**Pull request by zinccat**
Fixes #30

### CODE REVIEW

File: efax/_src/distributions/multivariate_normal/arbitrary.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
 @dataclass
 class MultivariateNormalNP(HasEntropyNP['MultivariateNormalEP'],
                            NaturalParametrization['MultivariateNormalEP', JaxRealArray],
-                           Multidimensional):
+                           Multidimensional,
+                            Samplable):
```
Comment (NeilGirdhar) [inline efax/_src/distributions/multivariate_normal/arbitrary.py:24]: Is there an extra space here? :smile:

File: efax/_src/distributions/multivariate_normal/arbitrary.py [hunk 2]
```diff
@@ -49,6 +50,23 @@ def log_normalizer(self) -> JaxRealArray:
         _, ld = jnp.linalg.slogdet(-self.negative_half_precision)
         return -0.25 * a - 0.5 * ld + 0.5 * self.dimensions() * jnp.log(np.pi)
 
+    @override
+    def sample(self, key: KeyArray, shape: Shape | None = None) -> JaxRealArray:
+        return self.to_variance_parametrization().sample(key, shape)
+
+    def variance(self) -> JaxRealArray:
```
Comment (NeilGirdhar) [inline efax/_src/distributions/multivariate_normal/arbitrary.py:57]: These are useful!  Thanks for adding them!

**Review by NeilGirdhar — COMMENTED** [review]
Thanks for doing this, just one minor comment

File: efax/_src/distributions/multivariate_normal/arbitrary.py (hunk 1 above)
Comment (zinccat) [reply efax/_src/distributions/multivariate_normal/arbitrary.py]: yeah, that's a weird indent, just fixed it

**Comment by NeilGirdhar** [comment]
Thank you!

**Pull request merged**